services/core-web/common/selectors/staticContentSelectors.js

```diff
@@ -321,6 +321,18 @@ export const getDropdownMineReportDefinitionOptions = createSelector(
   (options) => createDropDownList(options, "report_name", "mine_report_definition_guid")
 );
 
+export const getMineReportDefinitionHash = createSelector(
+  [getMineReportDefinitionOptions],
+  (options) =>
+    options.reduce(
+      (map, mine_report_definition) => ({
+        [mine_report_definition.mine_report_definition_guid]: mine_report_definition,
+        ...map,
+      }),
+      {}
+    )
+);
+
```

Comment:
```
export const getMineReportDefinitionHash = createSelector(
  [getDropdownMineReportCategoryOptions],
  createLabelHash
);
```

there is a function you can use to make the hash from the dropdown options.

Reply:
There should also be a test for this selector

Reply:
I looked at `createLabelHash` but it doesn't create the shape of object I need. It creates "string:string" when I needed "string:object".

Reply:
ohhh, okay I didn't notice that was an object. Okay.

Reply:
> There should also be a test for this selector

I made the test for this new selector :)

Reply:
👍